USERGUIDE.md

```diff
@@ -99,9 +110,9 @@ measure_ts,temperature
 (...)
 ```
 
-To start, head to the Nifi web interface, if you are using **minikube**, you can use the following command :
+To start, head to the Nifi web interface, type in your browser the `nifi.traefikIngress.host`. E.g. :
 ```
-minikube service -n fadi fadi-nifi
+http(s)://nifi.test.local
```

Comment:
should be a clickable url, not sure why the (s), I'd suggest to remove it if both work